rotkehlchen/assets/asset.py

```diff
@@ -85,6 +85,8 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     'COIN-2': 'COIN',
     strethaddress_to_identifier('0x15B543e986b8c34074DFc9901136d9355a537e7E'): 'STCCOIN',
     strethaddress_to_identifier('0x8f136Cc8bEf1fEA4A7b71aa2301ff1A52F084384'): 'STC',
+    'BCH': 'BCHN',
```

Comment:
This is the mapping missing right?

Reply:
no. Please read the issues carefully: https://github.com/rotki/rotki/issues/3354

This is the wrong exchange